javascript/javascript3/week3/visual/main.js

```diff
@@ -0,0 +1,34 @@
+async function translateOneByOne() {
+  await moveElement(document.querySelector("#red-circle"), {
+    x: 20,
+    y: 300
+  })
+  console.log("Red Element has been moved")
+  await moveElement(document.querySelector("#blue-circle"), {
+    x: 400,
+    y: 300
+  })
+  console.log("Blue Element has been moved")
+  await moveElement(document.querySelector("#green-circle"), {
+    x: 400,
+    y: 20
+  })
+  console.log("Green Element has been moved")
+}
+translateOneByOne()
+
+
+function translateAllAtOnce() {
+  Promise.all(moveElement(document.querySelector("#red-circle"), {
```

Comment:
The Promise.all() method takes an iterable of promises as an input. like  Promise.all([promise1,promise2,...]) you can read more about this here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
Got it👍🏻 Thanks for pointing it out